### PR TITLE
correct catalog task golangci lint path

### PR DESCRIPTION
**Pull request by pradeepitm12**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

File: tekton/release.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ git push --force ${PUSH_REMOTE} release-${RELEASE_VERSION}
 kubectl create namespace ${TARGET_NAMESPACE} 2>/dev/null || true
 
 for task in ${CATALOG_TASKS};do
-    kubectl -n ${TARGET_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/golang-${task}/0.1/golang-${task}.yaml
+    kubectl -n ${TARGET_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/golangci-${task}/0.1/golangci-${task}.yaml
```
Comment (piyush-garg) [inline tekton/release.sh:117]: for build and test, it is golang-*
for lint it is golangci-*

We need to handle this properly :D

Comment (chmouel) [reply tekton/release.sh:117]: since we removed lint i think we can actually remove the install of golangci

Comment (piyush-garg) [reply tekton/release.sh:117]: i see #1025 but it is still not merged

Comment (piyush-garg) [reply tekton/release.sh:117]: @pradeepitm12 If we get #1025 in, then we don't need this PR, all good

Comment (pradeepitm12) [reply tekton/release.sh:119]: oh that's nice, let's get @1025 in

**Comment by chmouel** [comment]
/lgtm

Looking Good! 🤙🏽

File: tekton/release.sh [hunk 2]
```diff
@@ -114,7 +114,12 @@ git push --force ${PUSH_REMOTE} release-${RELEASE_VERSION}
 kubectl create namespace ${TARGET_NAMESPACE} 2>/dev/null || true
 
 for task in ${CATALOG_TASKS};do
+  if [task == "lint"]
```
Comment (piyush-garg) [inline tekton/release.sh:117]: nit: it should be ${task}

**Pull request merged**